elements/demos/src/SingleSelectionDemo.js

```diff
@@ -0,0 +1,46 @@
+import SingleSelectionMixin from '../../elix-mixins/src/SingleSelectionMixin';
+import symbols from '../../elix-mixins/src/symbols';
```

Comment:
As referenced above, this is partly a documentation issue. Our plan is to ensure that developers working with Elix mixins understand the purpose and use of the standard properties and methods identified in the `symbols` module.